### ISSUE TITLE
Proposal: preview mode

### ISSUE THREAD

**Issue by alienalien13** [dmptrluke/django-markdownfield#10]
Would be great to preview. Since there is rendering included already, it doesn't sound hard to implement the `preview` feature. Seems to miss just a place(modal, whatever else) where this preview might be. WDYT? @benjaoming @dmptrluke @kpavlovsky 

Otherwise thanks for the tool!

**Comment by kostja-me** [comment]
Yeah, I think it is awesome. I must agree, that I have to save the page and go see how it looks every time. A tab just like the one I am typing here on GH should be quite enough.

**Comment by iamjameswalters** [comment]
If you use EasyMDE in your template, you'll have its preview built in. It's [not hard to get going](https://github.com/Ionaru/easy-markdown-editor#how-to-use). However, maybe there's a way to get it set up as a form widget? 

Edit: Nevermind...it already is 🤦‍♂️

**Comment by benjaoming** [comment]
I'm lucky to not have been asked for a preview function for this. I imagine that you can implement it many ways depending on the need.

I use django-markdownfield in combination with a lot of other fields that will all display a combined result, therefore I can't use an isolated frontend-only preview function. So even *if* there was something in the app, I would maybe want to disable it.

**Comment by dmptrluke** [comment]
This is currently being worked on (for admin only for now)
